2022q4/freebsd-foundation.adoc

```diff
@@ -4,54 +4,38 @@ Links: +
 link:https://www.FreeBSDfoundation.org[FreeBSD Foundation] URL: link:https://www.FreeBSDfoundation.org[https://www.FreeBSDFoundation.org] +
 link:https://freebsdfoundation.org/blog/technology-roadmap/[Technology Roadmap] URL: link:https://freebsdfoundation.org/blog/technology-roadmap/[https://FreeBSDFoundation.org/blog/technology-roadmap/] +
 link:https://www.FreeBSDfoundation.org/donate/[Donate] URL: link:https://www.FreeBSDfoundation.org/donate/[https://www.FreeBSDFoundation.org/donate/] +
-link:https://www.FreeBSDfoundation.org/FreeBSD-foundation-partnership-program/[Foundation Partnership Program] URL: link:https://www.FreeBSDfoundation.org/FreeBSD-foundation-partnership-program[https://www.FreeBSDFoundation.org/FreeBSD-foundation-partnership-program] +
+link:https://freebsdfoundation.org/our-donors/freebsd-foundation-partnership-program/[Foundation Partnership Program] URL: link:https://freebsdfoundation.org/our-donors/freebsd-foundation-partnership-program/[https://freebsdfoundation.org/our-donors/freebsd-foundation-partnership-program/] +
 link:https://www.FreeBSDfoundation.org/journal/[FreeBSD Journal] URL: link:https://www.FreeBSDfoundation.org/journal/[https://www.FreeBSDFoundation.org/journal/] +
 link:https://www.FreeBSDfoundation.org/news-and-events/[Foundation News and Events] URL: link:https://www.FreeBSDfoundation.org/news-and-events/[https://www.FreeBSDFoundation.org/news-and-events/]
```

Comment:
Why are you changing the capitalization for line 7 only?

@Jehops: Do you approve the capitalization change? If not, please note that the actual use of capital letters is inconsistent: there are occurences of "FreeBSDfoundation", "FreeBSDFoundation" and of "freebsdfoundation"

The issue regards the rest of the report as well. And it is also present in precedent reports, but now it is better to focus on this one.

Reply:
Looks good.

Reply:
Thanks @Jehops.

@grahamperrin: Could you please proceed to change all links in this report to remove the capitalization? I think you can leave the capitalization for the email addresses, just as we usually do for FreeBSD.org addresses.